sympy/polys/matrices/tests/test_sdm.py

```diff
@@ -376,35 +376,18 @@ def test_SDM_nullspace():
 
 
 def test_SDM_rref():
-    eye2 = SDM({0:{0:QQ(1)}, 1:{1:QQ(1)}}, (2, 2), QQ)
-
-    A = SDM({0:{0:QQ(1), 1:QQ(2)}, 1:{0:QQ(3), 1:QQ(4)}}, (2, 2), QQ)
-    assert A.rref() == (eye2, [0, 1])
-
-    A = SDM({0:{0:QQ(1)}, 1:{0:QQ(3), 1:QQ(4)}}, (2, 2), QQ)
-    assert A.rref() == (eye2, [0, 1])
-
-    A = SDM({0:{1:QQ(2)}, 1:{0:QQ(3), 1:QQ(4)}}, (2, 2), QQ)
-    assert A.rref() == (eye2, [0, 1])
-
-    A = SDM({0:{0:QQ(1), 1:QQ(2), 2:QQ(3)},
-             1:{0:QQ(4), 1:QQ(5), 2:QQ(6)},
-             2:{0:QQ(7), 1:QQ(8), 2:QQ(9)} }, (3, 3), QQ)
-    Arref = SDM({0:{0:QQ(1), 2:QQ(-1)}, 1:{1:QQ(1), 2:QQ(2)}}, (3, 3), QQ)
-    assert A.rref() == (Arref, [0, 1])
-
-    A = SDM({0:{0:QQ(1), 1:QQ(2), 3:QQ(1)},
-             1:{0:QQ(1), 1:QQ(1), 2:QQ(9)}}, (2, 4), QQ)
-    Arref = SDM({0:{0:QQ(1), 2:QQ(18), 3:QQ(-1)},
-                 1:{1:QQ(1), 2:QQ(-9), 3:QQ(1)}}, (2, 4), QQ)
-    assert A.rref() == (Arref, [0, 1])
-
-    A = SDM({0:{0:QQ(1), 1:QQ(1), 2:QQ(1)},
-             1:{0:QQ(1), 1:QQ(2), 2:QQ(2)}}, (2, 3), QQ)
-    Arref = SDM(
-            {0: {0: QQ(1,1)}, 1: {1: QQ(1,1), 2: QQ(1,1)}},
-            (2, 3), QQ)
-    assert A.rref() == (Arref, [0, 1])
+    # More tests are in test_rref.py
```

Comment:
I have moved all deleted tests to `test_rref` so that they can be tested against all implementations. Some were duplicated and so were just deleted from here but otherwise all tests were kept.